app/assets/stylesheets/dashboard.scss

```
@@ -165,6 +169,7 @@
     background: var(--theme-container-background, white);
     &.single-article-unpublished {
       background: lighten($yellow, 20%);
+      background: var(--theme-secondary-color, lighten($yellow, 20%));
```

```suggestion
      background: var(--theme-container-accent-background, lighten($yellow, 20%));
```

Let's change it to this variable. `--theme-secondary-color` is for font colors.

---

Problem I had with this was that it would change the background color of a draft to look the same as other articles. And then user had no real visual difference between a draft and a published article. Only the "draft" tag.

Should I still change it?

---

@venarius The suggested change by Ben is not the same as the other articles, published articles have background `--theme-container-background`

---

@Glennmen oh, yeah. You are right. My fault!